playground/mocks/data/idp/idx/email-challenge-consent.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
This is the `consent` screen remediation. A widget will need to be bootstrapped and rendered with this remediation when the user clicks on `Sign in` link from their email.